controls/button/configuration.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Content Alignment
+page_title: .NET MAUI Button Documentation - Content Alignment
+description: Learn how to align the content of the Telerik Button for .NET MAUI horizontally or vertically and apply image as a background.
+position: 2
+tags: .net maui, telerik .net maui, ui for .net maui, button, microsoft .net maui
+previous_url: /controls/button/content-alignment
+slug: button-configuration
+---
+
+# .NET MAUI Button Configuration
+
+
+The purpose of this help article is to show you the main configuration options of the control.
+
+## Content Alignment
```

Comment:
I suggest moving the section after `## Setting Image` because you must add the image first and then you can align the content.